recipe/meta.yaml

```diff
@@ -59,7 +59,7 @@ outputs:
         - wheel
       run:
         - {{ pin_subpackage("cachecontrol", exact=True) }}
-        - python 
+        - python
         - redis-py >=2.10.5
     test: *test
 
```

Comment:
[This skip](https://github.com/AnacondaRecipes/cachecontrol-feedstock/pull/4/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR68) as well.

Reply:
Are there other skips besides `skip: true # [py<36]` that I need to add?

Reply:
Never mind! I didn't see there were multiple. They're all good now.